src/entities/client.entity.ts

```diff
@@ -8,6 +8,7 @@ export interface OAuthClient {
   redirectUris: string[];
   allowedGrants: GrantIdentifier[];
   scopes: OAuthScope[];
+  [key: string]: any;
```

Comment:
We could make this change (it's conceptually equivalent, in my opinion, to https://github.com/jasonraimondi/ts-oauth2-server/blob/main/src/entities/user.entity.ts#L5).

However I don't **need** this change for my use-case, since I could do as you do in the prisma/express example https://github.com/jasonraimondi/ts-oauth2-server/blob/main/examples/prisma_express/src/entities/client.ts#L17, where you have an extra field defined on a class which implements the `OAuthClient` interface.

I've left this change in for now as a point for discussion.

Reply:
Personally I would probably always compose my classes as in the example, I have no problem at all adding this.